test/js-unit/metric-view-spec.js

```diff
@@ -4,151 +4,90 @@ var metricView = require( '../../src/disclosures/js/views/metric-view' );
+    metricView.metrics.test.school = 0.2;
+    metricView.metrics.test.better = 'lower';
+    var notificationClasses = metricView.getNotifications( 'test' );
     expect( notificationClasses ).to.equal( 'metric_notification__better' );
   });
 
-  it( 'gives the proper notification classes when the school value is higher than the national value (and that\'s a bad thing)', function() {
-    var schoolValue = parseFloat( 0.7 ),
-        nationalValue = parseFloat( 0.5 ),
-        sameMin = 0.4,
-        sameMax = 0.6,
-        betterDirection = 'lower',
-        notificationClasses = metricView.getNotificationClasses( schoolValue, nationalValue, sameMin, sameMax, betterDirection );
-    expect( notificationClasses ).to.equal( 'metric_notification__worse cf-notification cf-notification__error' );
-  });
-
-  it( 'gives the proper notification classes when the school value is lower than the national value (and that\'s a bad thing)', function() {
-    var schoolValue = parseFloat( 0.3 ),
-        nationalValue = parseFloat( 0.5 ),
-        sameMin = 0.4,
-        sameMax = 0.6,
-        betterDirection = 'higher',
-        notificationClasses = metricView.getNotificationClasses( schoolValue, nationalValue, sameMin, sameMax, betterDirection );
-    expect( notificationClasses ).to.equal( 'metric_notification__worse cf-notification cf-notification__error' );
-  });
-
-  it( 'gives the proper notification classes when sameMin and sameMax values are not numbers', function() {
-    var schoolValue = parseFloat( 0.3 ),
-        nationalValue = parseFloat( 0.5 ),
-        sameMin = parseFloat( '' ),
-        sameMax = parseFloat( '' ),
-        betterDirection = 'higher',
-        notificationClasses = metricView.getNotificationClasses( schoolValue, nationalValue, sameMin, sameMax, betterDirection );
+  it( 'gives the proper notification classes when the school value is ' +
+       'higher than the national value (and that\'s a bad thing)', function() {
+    metricView.metrics.test.school = 0.7;
+    metricView.metrics.test.better = 'lower';
+    var notificationClasses = metricView.getNotifications( 'test' );
+    expect( notificationClasses ).to.equal(
+      'cf-notification metric_notification__worse cf-notification__error'
+      );
+  });
+
+  it( 'gives the proper notification classes when the school value is ' +
+       'lower than the national value (and that\'s a bad thing)', function() {
+    metricView.metrics.test.national = 0.5;
+    metricView.metrics.test.school = 0.3;
+    var notificationClasses = metricView.getNotifications( 'test' );
+    expect( notificationClasses ).to.equal(
+      'cf-notification metric_notification__worse cf-notification__error'
+    );
+  });
+
+  it( 'gives the proper notification classes when min and max ' +
+       'values are not numbers', function() {
+    metricView.metrics.test.national = 0.5;
+    metricView.metrics.test.school = 0.5;
```

Comment:
Why did the school value increase here? Can we change this back to 0.3, to match the previous version of the test (refactoring it rather than testing something new)?